src/Button/Button.js

```diff
@@ -25,6 +25,7 @@ export const styleSheet = createStyleSheet('Button', (theme) => {
       color: palette.text.primary,
       backgroundColor: 'transparent',
       transition: transitions.multi(['background-color', 'box-shadow'], '250ms'),
+      willChange: 'transform',
```

Comment:
Actually, this works better, because that `transform: translateZ(0)` made texts blurry (didnt notice before) and this keeps texts as sharp as before.

BUT - it will also forces buttons to new layer (orange border in chrome dev tools), so i dont know if that performance hit still applies.

Also, problem is with **ALL buttons on page**, when that animation of InputField happen,